tests/suites/secrets_iaas/juju.sh

```diff
@@ -13,62 +9,68 @@ run_secrets_juju() {
 	wait_for "etcd" "$(idle_condition "etcd" 1 0)"
 	wait_for "active" "$(workload_status "etcd" 0).current"
 
+	echo "Apps deployed, creating secrets"
 	secret_owned_by_easyrsa_0=$(juju exec --unit easyrsa/0 -- secret-add --owner unit owned-by=easyrsa/0)
-	secret_owned_by_easyrsa_0_id=$(echo $secret_owned_by_easyrsa_0 | cut -d: -f 2)
+	secret_owned_by_easyrsa_0_id=${secret_owned_by_easyrsa_0##*/}
```

Comment:
Right, so do we need to worry about the compatibility issue of bringing the new URI format back to 3.1?

Reply:
no, uri is opaque